routes.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+)
+
+// Routes are named as "routeDOMAIN"
+
+func routeHome(writer http.ResponseWriter, req *http.Request) {
```

Comment:
Prefer the original parameter names **w** and **r** as this interface is quite common; keeping the standard names in common interfaces help to signal that you're compliant with the std.